File: test/ocamlTests/IntTest/isEvenTest.ml
test "isEven(8)" (fun () -> expect (Int.isEven 8) |> toEqual Eq.bool true) ; 
test "isEven(9)" (fun () -> expect (Int.isEven 9) |> toEqual Eq.bool false) ; 
test "isEven(0)" (fun () -> expect (Int.isEven 0) |> toEqual Eq.bool true) ; 

File: test/fsharpTests/IntTest/isEvenTest.fs
testCase "isEven(8)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Int.isEven 8) "error"
testCase "isEven(9)" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Int.isEven 9) "error"
testCase "isEven(0)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Int.isEven 0) "error"
